docs/source/release/v6.6.0/diffraction.rst

```diff
@@ -10,60 +10,64 @@ Powder Diffraction
+- Improve estimation of resolution parameters in :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>` to make it more robust to outliers.
+- Made Lorentz correction in :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>` optional.
+- Added back bank 14 to TOPAZ, for a total of 20 banks.
+- Exposed indexing tolerance for :ref:`SCDCalibratePanels  <algm-SCDCalibratePanels>` when using ``RecalculateUB`` option.
+- Sped up ellipsoid integration in :ref:`algm-IntegratePeaksMD` by introducing parallelization.
 
 Bugfixes
 ############
-- :ref:`IntegratePeaksProfileFitting <algm-IntegratePeaksProfileFitting>` library `ICCFitTools` module updated to support more recent versions of SciPy where the factorial function moved from `scipy.misc.factorial` to `scipy.special.factorial`. On newer versions of SciPy, the algorithm previously failed.
-- Replaced deprecated NumPy `np.int` type in :ref:`ConvertWANDSCDtoQ <algm-ConvertWANDSCDtoQ>`.
-- :ref:`SaveReflections <algm-SaveReflections>` now warns if an empty peak table is saved and doesn't fail with an error
-- Replaced deprecated Matplotlib `bivariate_normal` function in `BVGFitTools.py`
-- Fix issue where :ref:`LoadIsawPeaks <algm-LoadIsawPeaks>` fails when loading peaks with modulation vectors that cannot recover modulated UB matrix.
-- Ensuring colorscale max > min (found for weak peaks with pixels having 0 intensity) in :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>`.
-- :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>` will now close a generated figure if an error occured while trying to write to file (e.g. file of same name was already open)
-- Fix bug that didn't set HKL of peak if ``UpdatePeakPosition = True`` in :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>`
-- Fix bugs in :ref:`SelectCellWithForm <algm-SelectCellWithForm>` and ref:`SelectCellOfType <algm-SelectCellOfType>` that incorrectly transform modulation vectors.
-- Fix bug where modulation vectors are not transformed in :ref:`TransformHKL <algm-TransformHKL>`.
+- The :ref:`algm-IntegratePeaksProfileFitting` library ``ICCFitTools`` module has been updated to support more recent versions of SciPy where the factorial function moved from ``scipy.misc.factorial`` to ``scipy.special.factorial``. On newer versions of SciPy, the algorithm previously failed.
+- Replaced deprecated NumPy ``np.int`` type in :ref:`ConvertWANDSCDtoQ <algm-ConvertWANDSCDtoQ>`.
+- :ref:`algm-SaveReflections` now warns if an empty peak table is saved and doesn't fail with an error.
+- The deprecated Matplotlib ``bivariate_normal`` function has been replaced in ``BVGFitTools.py``.
+- :ref:`LoadIsawPeaks <algm-LoadIsawPeaks>` no longer fails when loading peaks with modulation vectors that cannot recover the modulated UB matrix.
+- Ensuring colorscale max > min (found for weak peaks with pixels having 0 intensity) checks have been added to :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>`.
```

Comment:
I wondered if something like this would help the flow of this sentence a bit:

> Checks to ensure colorscale max > min (found for weak peaks with pixels having 0 intensity) have been added to :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>`.